recipes/libzippp/all/conandata.yml

```diff
@@ -1,4 +1,7 @@
 sources:
+  "5.0":
```

Comment:
I agree with you about the versioning.
`5.0-1.8.0` is the real version.

Reply:
I changed the version name from `5.0` to `5.0-1.8.0`.